include/drivers/console/bm_serial.h

```diff
@@ -26,6 +26,8 @@ extern "C" {
 #define MAX_ENCODED_BUF_SIZE	258
 #define TASK_STACK_SIZE         1024
 
+#define MAX_SERIAL_DEV_COUNT    3
```

Comment:
Let's change the naming here to MAX_BM_SERIAL_DEV_COUNT. I could see this one potentially having conflicts with other zephyr or user code as is.

Reply:
In the same vein, we should probably be consistent about applying such pre-fixes across the board within the BM drivers and codebase.